src/main/java/io/reactivex/disposables/Disposable.java

```diff
@@ -23,6 +23,7 @@
 
     /**
      * Returns true if this resource has been disposed.
```

Comment:
Yeah I know that but then `* Returns true if this resource has been disposed.` could be removed, could not it?